pkg/render/manager_test.go

```diff
@@ -695,9 +703,13 @@ var _ = Describe("Tigera Secure Manager rendering tests", func() {
 	// renderManager passes in as few parameters as possible to render.Manager without it
 	// panicing. It accepts variations on the installspec for testing purposes.
 	renderManager := func(i *operatorv1.InstallationSpec) *appsv1.Deployment {
+		var esConfigMap *relasticsearch.ClusterConfig
+		if !operatorv1.IsFIPSModeEnabled(i.FIPSMode) {
```

Comment:
It's not obvious why we pass this in only for FIPS disabled - a comment would help here.

Reply:
True, I'll port the comment I had in the controller here as well.